Create CSS Entries for translateX & translateY and Update translate (#576)
Codecademy/docs: content/css/concepts/transform-functions/terms/translateX/translateX.md

---
Title: 'translateX()'

> Description: 'Translates an element along the X axis.'

Reviewer: ```suggestion
Description: 'Translates an element along the X-axis.'
```

> Description: 'Translates an element along the X-axis.'

Reviewer 2: ```suggestion
Description: 'Moves an element horizontally along the X-axis.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Functions'
  - 'Positioning'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
  - 'paths/full-stack-engineer-career-path'
---

> The `translateX` function translates an element along the X axis.

Reviewer: ```suggestion
The `translateX` function moves an element along the horizontal X-axis.
```

> The `translateX` function moves an element along the horizontal X-axis.

Reviewer 2: ```suggestion
The `translateX()` function moves an element along the horizontal X-axis.
```

## Syntax

```pseudo
transform: translateX(<value>);
```

The required `<value>` can be one of the following:

- Length value: `100px`, `1.5em`
- Percentage value: `25%`, `50%`

> A positive value moves an element right along the horizontal axis while a negative value will move an element left.

Reviewer: ```suggestion
A positive value moves an element towards the right along the horizontal X-axis, while a negative value will move an element left.
```

> Note: `translateX(<value>)` is functionally equivalent to `translate(<value>)` when [`translate`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) is called with a single value.

Reviewer 2: ```suggestion
**Note**: `translateX(<value>)` is functionally equivalent to `translate(<value>)` when [`translate`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) is called with a single value.
```

## Example 1

> Move the `.box` element `100px` right:

Reviewer: ```suggestion
Move the element, with a `class` of `box`, `100px` to the right:
```

> Move the element, with a `class` of `box`, `100px` to the right:

Reviewer 2: ```suggestion
In the example below, an element with a `.box` class is moved `100px` to the right:
```

```css
.box {
  transform: translateX(100px);
}
```

## Example 2

> Move the `.box` element `10em` left:

Reviewer 2: ```suggestion
In this example, the element is moved to the left by `10em`, relative to the width of its direct parent:
```
